src/checks_util.cc

```diff
@@ -282,15 +289,19 @@ zetasql_base::StatusOr<VisitResult> RuleVisitor::defaultVisit(
   return VisitResult::VisitChildren(node);
 }
 
-std::vector<ParseToken> GetKeywords(absl::string_view sql) {
+std::vector<ParseToken> GetKeywords(absl::string_view sql, ErrorCode code) {
```

Comment:
Oh, this ErrorCode is really confusing here. Should we rename it to LintCheckCode or something? (Not in this pull request, please :)

Reply:
Hmm, it is ErrorCode inside LintError, but it also correspond to check (each check has different error code.) So, yes, we can rename it, but it is a very core class.

Reply:
Yes, I understand what it means, but I scratched my head for 30 seconds, before I figured this out :) Let's rename in a separate cleanup PR.